devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus-jbang/code/jbang-picocli-code/base/README.tpl.qute.md

```diff
@@ -0,0 +1,35 @@
+## Greet the world!
```

Comment:
Maybe we can make the doc a bit simpler by showing one command to say hello and another to show the help.